developer-docs-site/docs/concepts/staking.md

```diff
@@ -9,18 +9,14 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 ## Concept
 
-:::tip
+:::tip Consensus
 We strongly recommend that you read the consensus section of the [Life of a Transaction](/guides/basics-life-of-txn#consensus) before proceeding further. 
 :::
 
 In a distributed system like blockchain, executing a transaction is different from updating the state of the ledger and persisting the results in storage. An agreement, i.e., consensus, must be reached by a quorum of validators on the ordering of transactions and their execution results before the results are persisted in storage and the state of the ledger is updated. 
```

Comment:
"In a distributed system like blockchain, executing a transaction is different from updating the state of the ledger and persisting the results in storage"

=> I'm not sure if I understand this sentence. Is this saying that executing the transaction is _**not as simple as just**_ updating storage?

Reply:
It is saying the execution of the transaction is separate/distinct from updating the state of the ledger. I want to make it clear that execution of a transaction does not automatically mean that the transaction results will go into the storage. For example in step 11 here https://aptos.dev/guides/basics-life-of-txn#executing-the-block-and-reaching-consensus it says:
 _"...the consensus leader attempts to reach consensus on the proposed block's execution result with the other validator nodes participating in consensus."_

Reply:
I have in my todo list to clarify that life of txn document, later on.